harness/engine/profiles.go

```diff
@@ -64,7 +66,7 @@ func Profile1(index int, node *DuskNode, walletPath string) {
 	viper.Set("timeout.timeoutkeepalivetime", 30)
 
 	viper.Set("logger.output", node.Dir+"/dusk")
-	viper.Set("logger.level", "info")
+	viper.Set("logger.level", "trace")
```

Comment:
Maybe a typo? we should have `info` here, right?

Reply:
I swapped this in order to debug some things and accidentally committed it. Will revert.